recipes/curl/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "7.48.0" %}
+
+package:
+  name: curl
+  version: {{ version }}
+
+source:
+  fn: curl-{{ version }}.tar.bz2
+  url: http://curl.haxx.se/download/curl-{{ version }}.tar.bz2
+  sha256: 864e7819210b586d42c674a1fdd577ce75a78b3dda64c63565abe5aefd72c753
+
+build:
+  number:  0
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    # python is a build requirement on Windows to resolve features.
+    - python      # [win]
+    - pkg-config  # [unix]
+    - openssl     # [unix]
+    - certifi     # [unix]
+    - zlib
+  run:
+    - openssl     # [unix]
```

Comment:
Probably need a way to activate feature here, too.  This package is tricky because it is *both* a library (libcurl) and an executable.

I recommend including python - probably don't need the jinja here though.  Might not need it in tests if you add it here.
Could use python